packages/cozy-client/src/models/doctypes/index.js

```diff
@@ -0,0 +1,3 @@
+import * as locales from './locales'
```

Comment:
<img width="996" alt="Capture d’écran 2022-07-28 à 11 52 04" src="https://user-images.githubusercontent.com/8363334/181477142-09458f4f-fd9e-4081-a0c6-535b38e00b82.png">

Are we sure this is working?

tests are failing

looks like we should do something like

import * as locales from `./locales/${lang}`

Reply:
We should create an index.js file inside ./locales that export local by local

Reply:
done